app-creator/client/src/utils/helpers.ts

```diff
@@ -207,3 +214,85 @@ export function removeBackgroundColorFromSharedWidget(element: HTMLElement) {
   elementStyle.padding = elementStyle.padding ?? sharedAttributes.padding.value;
   (element as EEWidget).setStyle(elementStyle);
 }
+
+/**
+ * Get sample code snippet for creating an app on the code editor.
```

Comment:
We need this to be a function rather than a constant since it injects the template inside.